### DISCUSSION TITLE
[FFmpeg-devel] [PATCH 2/4] avcodec/nvenc: Add intra refresh support

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#187] [General]
You guys are probably aware of these recent commits to FFMPEG, but just wanted to make sure.   Seems there has been some work to enable intra-refresh among other things for NVenc that might benefit us here.

Thanks,
Alan

http://ffmpeg.org/pipermail/ffmpeg-devel/2021-September/284683.html
http://ffmpeg.org/pipermail/ffmpeg-devel/2021-September/284684.html

**Comment by MartinPulec** [comment]
Yes, thanks - we know about it. We used to patch FFmpeg in UltraGrid AppImage by ourselves (since that changes I've dropped the custom patches) so in this particular case there is no change but other builds (namely Windows) may utilize this.
